scripts/update_schemas.py

```diff
@@ -295,7 +178,7 @@
     basename = schema + "." + version + ".json"
     zip_filepath = os.path.join(VERSION, VERSION, "json-schema", basename)
     schemadir = os.path.join(json_schema_path, schema)
-    os.makedirs(schemadir)
+    os.makedirs(schemadir, exist_ok=True)
```

Comment:
Why add this?

Reply:
without that it now overwriting that folder (we delete that file), because the folder is still there even we delete all files.

Reply:
I was getting errors, now I tried that without `exist_ok=True` that error is gone that mean we are deleting that folder now

so let me remove that.